src/main.js

```diff
@@ -122,83 +115,77 @@ var quotes = [
 var savedPosters = [];
 var currentPoster;
 
```

Comment:
As you get into larger files, it can be helpful to label your sections with comments (as mentioned above) like `//////// GLOBAL VARIABLES`, `/////// EVENT LISTENERS` etc.